### ISSUE TITLE
Syntax Error chroniker_tags.py line 34: python 3.4.2 

### ISSUE THREAD

**Issue by robitnik** [chrisspen/django-chroniker#32]
Raised syntax error on line 34: chroniker_tags.py. On a clean install with python 3.4.2.

I edited the line as follows:
```
  try:
        # Splitting by None == splitting by spaces.
        tag_name, object_id = token.contents.split(None, 1)
    except ValueError:
       # line below causing syntax error python 3.4.2
       # raise template.TemplateSyntaxError, "%r tag requires one argument" % token.contents.split()[0]
       raise template.TemplateSyntaxError
    return RunJobURLNode(object_id)
```

**Comment by chrisspen** [comment]
Thanks, I've confirmed this. I don't formally support 3.4, but I'll add it to my test harness for the next release.

**Comment by chrisspen** [comment]
I've committed a fix for this in the 0.8.0 release.
